dts/riscv/gigadevice/gd32vf1.dtsi

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "nuclei,bumblebee", "riscv";
+			reg = <0>;
+			riscv,isa = "rv32imac";
+		};
+	};
+
+
+	sram0: memory@20000000 {
+		compatible = "mmio-sram";
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "gd,gd32vf103-soc", "simple-bus";
+		ranges;
+
+		eclic: interrupt-controller@d2000000 {
+			#interrupt-cells = <2>;
+			compatible = "nuclei,eclic";
+			interrupt-controller;
+			reg = <0xd2000000 0x0001
+			       0xd2000004 0x0004
+			       0xd200000b 0x0001
+			       0xd2001000 0x1000>;
+			label = "NUCLEI_ECLIC";
+		};
+
+		flash: flash-controller@40022000 {
+			compatible = "gd,gd32-flash-controller";
+			label = "FLASH_CTRL";
+			reg = <0x40022000 0x400>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			flash0: flash@8000000 {
+				compatible = "soc-nv-flash";
+				label = "FLASH_GD32";
+
+				write-block-size = <2>;
+			};
+		};
+
+		usart0: serial@40013800 {
+			compatible = "gd,gd32-usart";
+			reg = <0x40013800 0x400>;
+			interrupts = <37 0>;
+			interrupt-parent = <&eclic>;
+			rcu-periph-clock = <0x60e>;
+			status = "disabled";
+			label = "UART_0";
+		};
+
+		usart1: serial@40004400 {
+			compatible = "gd,gd32-usart";
+			reg = <0x40004400 0x400>;
+			interrupts = <38 0>;
+			interrupt-parent = <&eclic>;
+			rcu-periph-clock = <0x711>;
+			status = "disabled";
+			label = "UART_1";
+		};
+
+		usart2: serial@40004800 {
+			compatible = "gd,gd32-usart";
+			reg = <0x40004800 0x400>;
+			interrupts = <39 0>;
+			interrupt-parent = <&eclic>;
+			rcu-periph-clock = <0x712>;
+			status = "disabled";
+			label = "UART_2";
+		};
+
+		afio: afio@40010000 {
+			compatible = "gd,gd32-afio";
+			reg = <0x40010000 0x400>;
+			rcu-periph-clock = <0x600>;
+			status = "okay";
+			label = "AFIO";
+		};
+
+		pinctrl: pin-controller@40010800 {
+			compatible = "gd,gd32-pinctrl-afio";
+			reg = <0x40010800 0x1c00>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			status = "okay";
+			label = "PINCTRL";
+
+			gpioa: gpio@40010800 {
+				compatible = "gd,gd32-gpio";
```

Comment:
nit: add missing
```
gpio-controller;
#gpio-cells = <2>;
```

Reply:
Same as above comment.

Reply:
I'll apply it.
And the unavailable gpiof and gpiog entries have been added, so delete them.